trace_replay/block_cache_tracer.h

```diff
@@ -66,6 +67,8 @@ struct BlockCacheLookupContext {
   // how many blocks a Get/MultiGet request accesses. We can also measure the
   // impact of row cache vs block cache.
   uint64_t get_id = 0;
+  std::string referenced_key;
+  bool is_snapshot_get;
```

Comment:
Can you remind me of why distinguish between snapshot is nullptr and not nullptr?

Reply:
This is for evaluating row cache. We construct the row key as "sst file number" + "user key" + "snapshot id". The snapshot id is 0 if it is not a snapshot get. 
https://github.com/facebook/rocksdb/blob/master/db/table_cache.cc#L273